.vscode/tasks.json

```diff
@@ -9,7 +9,8 @@
         {
             "label": "v5_hal Firmware Build",
             "type": "shell",
-            "command": "./scripts/run_docker.ps1 'catkin_make v5_hal_firmware_build'",
+            "windows": { "command": "./scripts/run_docker.ps1 'catkin_make v5_hal_firmware_build'" },
+            "linux": { "command": "pwsh ./scripts/run_docker.ps1 'catkin_make v5_hal_firmware_build'" },

```

Comment:
We might want to add the pwsh apt package to the install script (just in case people don't have it and start getting strange warnings)

Reply:
Added pwsh (Powershell) to devInstallScript in dc4fea7.